Bindings/Laerdal.Maui.AzureCommunicationCalling.iOS/Laerdal.Maui.AzureCommunicationCalling.iOS.csproj

```diff
@@ -1,49 +1,48 @@
-﻿ <Project Sdk="Microsoft.NET.Sdk">
-   <PropertyGroup>
-     <PackageId>Laerdal.Maui.AzureCommunicationCalling.iOS</PackageId>
-     <version>2.3.0-beta.5.1</version>
-     <TargetFrameworks>net6.0-ios</TargetFrameworks>
-     <Nullable>enable</Nullable>
-     <ImplicitUsings>true</ImplicitUsings>
-     <IsBindingProject>true</IsBindingProject>
-     <AssemblyName>Laerdal.Maui.AzureCommunicationCalling.iOS</AssemblyName>
-     <Title>AzureCommunicationCalling for net6.0-ios</Title>
-     <Authors>Laerdal Medical</Authors>
-     <PackageReadmeFile>Readme.md</PackageReadmeFile>
-     <Description>
-Binding for Azure Communication Calling SDK for iOS.
+﻿<Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
+        <PackageId>Laerdal.Maui.AzureCommunicationCalling.iOS</PackageId>
+        <version>2.8.0.8</version>
+        <TargetFramework>net7.0-ios</TargetFramework>
```

Comment:
Is there a good reason not to jump on the net8 wagon, seeing as this is released and LTS?

Reply:
I tried but I encountered some issues.
Maybe this can be done later in separate PR?